debian/changelog

```diff
@@ -1,3 +1,10 @@
+fossology (3.9.0-1) unstable; urgency=low
+
+  * New upstream release
+  * https://github.com/fossology/fossology/releases/tag/3.9.0 for details
+
+ -- Gaurav Mishra <mishra.gaurav@siemens.com>  Mon, 30 Nov 2020 11:11:11 +0530
```

Comment:
interesting date ... r u planning to wait or is just a guess?

Reply:
I just left the date so we can do tests on master before tagging. If no issues were found, I will change the date before tagging.